src/v8/mikro-orm-dotenv.ts

```diff
@@ -0,0 +1,18 @@
+import { existsSync } from "fs";
+import { readFile, writeFile } from "fs/promises";
+
+/**
+ * Add a dotenv call to config.
+ * See https://mikro-orm.io/docs/upgrading-v5-to-v6#env-files-are-no-longer-automatically-loaded.
+ */
+export default async function addDotenvCallToConfig() {
+    if (!existsSync("api/src/db/ormconfig.cli.ts")) {
+        return;
+    }
+
+    let fileContent = await readFile("api/src/db/ormconfig.cli.ts", "utf-8");
+
+    fileContent = `import "dotenv/config";\n\n${fileContent}`;
```

Comment:
are you sure we need this? we do use dotenv already "somewhere"

Reply:
It's what MikroORM suggests in their upgrade guide. Let's discuss this here: https://github.com/vivid-planet/comet/pull/2822#discussion_r1891211910